src/scripts/modules/games.js

```diff
@@ -0,0 +1,196 @@
+import PokeLogo from '../../assets/icons/pokeLogo.svg';
+
+const shuffleCards = (cardsArray) => { // Fisher-Yates Shuffle Algorithm.
+  for (let i = cardsArray.length - 1; i > 0; i -= 1) {
+    const randIndex = Math.floor(Math.random() * (i + 1));
+    cardsArray[randIndex].style.order = i;
+    cardsArray[i].style.order = randIndex;
+  }
+};
+
+const getCardType = (card) => card.children[0].children[0].children[0].src;
```

Comment:
We could add a class to the ```img``` tag so it would be more readable to retrieve the src of the image

Reply:
Done!